src/realm.h

```diff
@@ -2025,7 +2026,8 @@ RLM_API void realm_collection_changes_get_ranges(
  * @param out_modification_size number of modifications
  */
 RLM_API void realm_dictionary_get_changes(const realm_dictionary_changes_t* changes, size_t* out_deletions_size,
-                                          size_t* out_insertion_size, size_t* out_modification_size);
+                                          size_t* out_insertion_size, size_t* out_modification_size,
+                                          bool* out_was_deleted);
```

Comment:
We need to update the documentation as well. Can you add the argument above in the comment pls?